js/src/util/scrollbar.js

```diff
@@ -49,7 +49,7 @@ const reset = () => {
 const _resetElementAttributes = (selector, styleProp) => {
   SelectorEngine.find(selector).forEach(element => {
     const value = Manipulator.getDataAttribute(element, styleProp)
-    if (typeof value === 'undefined' && element === document.body) {
+    if (typeof value === 'undefined') {
```

Comment:
I was just thinking if the value was not stored on the data attribute then why the style prop it is being removed from the element? :thinking:

Reply:
you are right on this